src/js/core/mejs.js

```diff
@@ -31,7 +31,7 @@ mejs.html5media = {
 	 * @type {String[]}
 	 */
 	methods: [
-		'load', 'play', 'pause', 'canPlayType'
+		'load', 'play', 'pause', 'canPlayType','canplay'
```

Comment:
Per comments in previous ticket, this is not a method but an event. This is not valid